_posts/pl/2021-04-30-webpack-externals-vs-bundled.md

```diff
@@ -0,0 +1,157 @@
+---
+layout:    post
+title:     "Microfrontends: czy współdzielić zależności?"
```

Comment:
Jak to w końcu jest: micro frontends czy microfrontends? Widzę że raz się pisze tak, raz tak więc pewnie obie są poprawne i nie jest to uregulowane przez jakąś wykładnię językową. Porównałem więc, która forma jest częściej wpisywana w Google i wychodzi na to, że ta pisana rozłącznie.

Reply:
Fowler pisze rozłącznie to my też tak piszmy :)

https://martinfowler.com/articles/micro-frontends.html

Reply:
👍